src/Apps/Order2/Order2App.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, type BoxProps, breakpoints } from "@artsy/palette"
+import { useFlag } from "@unleash/proxy-client-react"
+import { ErrorPage } from "Components/ErrorPage"
+import { useSystemContext } from "System/Hooks/useSystemContext"
+import type * as React from "react"
+import { Meta } from "react-head"
+import styled from "styled-components"
+
+export const Order2App: React.FC<React.PropsWithChildren> = ({ children }) => {
+  const isRedesignEnabled = useFlag("emerald_checkout-redesign")
+  const { isEigen } = useSystemContext()
+
+  // TODO: This renders briefly before the real page appears (checkout & details)
+  if (!isRedesignEnabled) return <ErrorPage code={404} />
+
+  return (
+    <>
+      <Meta
+        name="viewport"
+        content={
+          isEigen
+            ? "width=device-width, user-scalable=no"
+            : "width=device-width, initial-scale=1, maximum-scale=5 viewport-fit=cover"
+        }
+      />
+      <SafeAreaContainer>
+        <AppContainer>{children}</AppContainer>
+      </SafeAreaContainer>
+    </>
```

Comment:
#blocking - We should stick with modern Force conventions, and keep things fresh without copying from old app, which is ancient.  The route config has a `layout` prop for going full screen with logo, and so on. But in either case, just something simple like:

```tsx
<Box width={600}>
  {children}
</Box>
``` 

Is fine for doing static layouts (as seen in the vanilla `checkout` [app](https://github.com/artsy/force/blob/main/src/Apps/Checkout/CheckoutApp.tsx)).

Reply:
Ah, thanks. Should I remove the Meta & Head components I've copied over as well?

Reply:
Yep. As we talked about yesterday, it will be helpful to look around force a bit and familiarize yourself with some patterns before continuing, or reach out further and I can point you at some good examples 👍